misc/e2elive.py

```diff
@@ -83,12 +83,16 @@ def main():
         from botocore.config import Config
         from botocore import UNSIGNED
         s3 = boto3.client("s3", config=Config(signature_version=UNSIGNED))
+        prefix = "indexer/e2e4"
         if "/" in tarname:
-            tarname = tarname.split("/")[1]
+            cmhash_tarnme = tarname.split("/")
+            cmhash = cmhash_tarnme[0]
+            tarname =cmhash_tarnme[1]
+            prefix+="/"+cmhash
             tarpath = os.path.join(tempdir, tarname)
```

Comment:
Could you elaborate on whats happening here?

If I'm reading right it looks like there were multiple slashes in the name and this supports a tarname with 1 slash. Should we support multiple slashes?

Reply:
I don't think it's possible to support filenames with `/` without having to update the prefix.